### PR TITLE
define __all__

### PR DESCRIPTION
**Pull request by gkorland**
define __all__  to make sure `from .util import *` won't polute the globals

### CODE REVIEW

File: redisgraph/util.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import random
 import string
 
+__all__ = ['random_string', 'quote_string']
```
Comment (swilly22) [inline redisgraph/util.py:4]: Why is this necessary ?
when importing everything from this module (util.py) the importer will get `random_string` and `quote_string` by default.

**Review by swilly22 — CHANGES_REQUESTED** [review]

File: redisgraph/util.py (hunk 1 above)
Comment (gkorland) [reply redisgraph/util.py:4]: @swilly22 https://lgtm.com/projects/g/RedisGraph/redisgraph-py/alerts/?mode=list

**Review by swilly22 — APPROVED** [review]
